fuzzers/030-iob/Makefile

```diff
@@ -12,9 +12,11 @@ build/segbits_xiob33.db: build/segbits_xiob33.rdb process_rdb.py bits.dbf
 	${XRAY_MASKMERGE} build/mask_xiob33.db $$(find -name segdata_liob33.txt) $$(find -name segdata_riob33.txt)
 
 pushdb:
+ifneq (${XRAY_DATABASE}, zynq7)
```

Comment:
Just remove the ifneq? You added them as part of making things green, but with the change to generate.py, things should be okay with them?

Reply:
Because for Zynq we only have RIOBs? There is the PS on the left instead of LIOBs.

Reply:
> Because for Zynq we only have RIOBs? There is the PS on the left instead of LIOBs.
This is true only for smaller Zynq's. Bigger ones (kintex based) have RIOB's, but we don't support those now.

Reply:
Agreed. However currently the fuzzer assumes that the bits for liob are identical with the ones for riob which will probably not be true for Zynqs. This will have to be adressed once we start supporting other Zynqs but currently it is not true so why add this to the database.